RFCs/0011-partiql-insert-update-spec.md

```diff
@@ -185,9 +185,11 @@ Optional attributes in target schema that have the same name in the resulting tu
 
 In the following examples, `Films` table has a closed schema. In the table `len` attribute is implicitly defined as nullable.
 
+_In the following examples `CREATE TABLE` including `SCHEMA OPEN` and `SCHEMA CLOSED` syntax is arbitrary since the PartiQL DDL is yet to be defined._
+
```

Comment:
*Minor*: I would define what these terms mean.  Here is what I might write for this:

`SCHEMA OPEN` means that the struct/tuple may have attributes not defined in the schema and allows for defined attributes to be _optional_. `SCHEMA CLOSED` means that the attributes defined must all be present and there is no _optional_ attributes (i.e. all attributes MUST have a value, even if that value is `NULL`).